softwareComponents/rust-rofi_voxel/src/pos/mod.rs

```diff
@@ -2,3 +2,20 @@ pub type IndexType = u8;
 
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct VoxelPos(pub [IndexType; 3]);
+
+pub fn compute_minimal_pos_hull<TIndex, IPos>(
+    positions: IPos,
+) -> Option<[std::ops::Range<TIndex>; 3]>
```

Comment:
Kdy nemá tělo minimální obal?

Reply:
Když nemám žádnou pozici

Reply:
A takové konfigurace máš?

Reply:
Přijde mi víc Rustové to kontrolovat v kusu kódu, kde se starám o to, že to není prázdné - a u funkce předpokládat co nejmíň věcí, pokud mě to moc nestojí.